src/token-group/__tests__/token-group.test.tsx

```diff
@@ -169,4 +185,64 @@ describe('TokenGroup', () => {
       expect(icon!.getElement()).toContainHTML(icons['treeview-collapse']);
     });
   });
+
+  describe('Focus management', () => {
+    test('Focus is dispatched to the next active token when non-last token is removed', () => {
+      const wrapper = renderStatefulTokenGroup({
+        items: [
+          { label: '1', dismissLabel: 'Remove 1' },
+          { label: '2', dismissLabel: 'Remove 2' },
+          { label: '3', dismissLabel: 'Remove 3', disabled: true },
+          { label: '4', dismissLabel: 'Remove 4' },
+        ],
+      });
+      act(() => wrapper.findToken(2)!.findDismiss().click());
```

Comment:
This `act` does nothing, after https://github.com/cloudscape-design/test-utils/pull/25

Why did you add it?

Reply:
Thanks, Boris - I'll check.